client/karma.js

```diff
@@ -112,7 +112,12 @@ var Karma = function (socket, iframe, opener, navigator, location) {
     if (url) {
       message = msg + '\nat ' + url + (line ? ':' + line : '')
     }
-
+    
+    // create an object with the string representation of the message to ensure all its content is properly 
+    // transferred to the console log
+    message = {message: message, str: message.toString().replace(/\t/g, "    ").split("\n")}
```

Comment:
the replace and split parts are for nicer output display in the console, since the object is converted to json. 
With the split and replace, you get 
```
"str": [
      "Error: (SystemJS) Object prototype may only be an Object or null: undefined",
      "    TypeError: Object prototype may only be an Object or null: undefined",
      "        at setPrototypeOf (<anonymous>)",
      "        at __extends$1 (dist/bundles/my-bundle.umd.js:2258:9)",
    ]
```
instead of 

```
"str": "Error: (SystemJS) Object prototype may only be an Object or null: undefined\n\tTypeError: Object prototype may only be an Object or null: undefined\n\tat setPrototypeOf (<anonymous>)\n\tat __extends$1 (dist/bundles/my-bundle.umd.js:2258:9)",
```